addons/frag/functions/fnc_dev_clearTraces.sqf

```diff
@@ -0,0 +1,23 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Cleares all dev spheres and traces
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_frag_fnc_dev_clearTraces;
+ *
+ * Public: No
+ */
+for "_i" from 0 to count GVAR(dev_eventSpheres) - 1 do {
+    deleteVehicle (GVAR(dev_eventSpheres)#_i);
+};
```

Comment:
```suggestion

{
    deleteVehicle _x;
} forEach GVAR(dev_eventSpheres);
```

Reply:
t's a rare call, for debugging, and the performance difference isn't too great, but sure!